web/packages/teleport/src/Discover/SelectResource/SelectResource.tsx

```diff
@@ -608,6 +625,18 @@ const Grid = styled.div`
   row-gap: 15px;
 `;
 
+const ExternalLinkIconWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  justify-content: flex-end;
+  position: absolute;
+  top: 12px;
+  right: 12px;
```

Comment:
I'd use a standardized spacing values from theme (16px):
```suggestion
  top: ${props => props.theme.space[3]}px;
  right: ${props => props.theme.space[3]}px;
```